web/src/components/DatePicker.tsx

```diff
@@ -75,16 +41,22 @@ const isValidJsDate = (date: Date | null): boolean => {
   if (date == null) {
     return false
   }
-  try {
-    const checkDate = DateTime.fromJSDate(date)
-    const maxValidYear = 2500
-    // Added this operator to prevent crashing rrule when typing random number
-    return checkDate.year <= maxValidYear
-  } catch (_) {
-    return false
+  const checkDate = DateTime.fromJSDate(date)
+  const maxValidYear = 2500
+  return checkDate.year <= maxValidYear
+}
```

Comment:
❓ Is the `isValidJsDate` still necessary? I think we already get a luxon `DateTime` so imo we can just set the date directly 🤔

Reply:
Can we use `maxDate` instead? https://mui.com/x/react-date-pickers/date-picker/#order-of-years

Reply:
Nice find thanks.